src/app/core/_components/forms/custom-forms/superhashlist/new-superhashlist/new-superhashlist.component.html

```diff
@@ -0,0 +1,27 @@
+<grid-main>
+
+  <app-page-subtitle [subtitle]="'Create Superhaslist'"></app-page-subtitle>
+
+  <form [formGroup]="form" (ngSubmit)="onSubmit()">
+
+    <app-input-text title="Name" formControlName="name" [error]="error?.name"></app-input-text>
+
+    <app-input-multiselect
+      [isLoading]="isLoading"
+      [placeholder]="'Haslists...'"
+      [arrayOfObjects]="selectHashlists"
+      [mergeIdAndName]="true"
+      [label]="'Select or search Hashlists:'"
+      [externalControl]="form.get('hashlistIds')"
```

Comment:
Hmm, what is the purpose of this externalControl? Can't you just pass `form` ?

Reply:
Just remove it, was not need it